bigquery/google/cloud/bigquery/job.py

```diff
@@ -131,13 +131,13 @@ class DestinationFormat(object):
 class Encoding(object):
     """The character encoding of the data. The supported values
     are `UTF_8` corresponding to `'UTF-8'` or `ISO_8859_1` corresponding to
-    `'ISO-8559-1'`. The default value is `UTF_8`.
+    `'ISO-8859-1'`. The default value is `UTF_8`.
 
     BigQuery decodes the data after the raw, binary data has been
     split using the values of the quote and fieldDelimiter properties.
     """
     UTF_8 = 'UTF-8'
-    ISO_8559_1 = 'ISO-8559-1'
+    ISO_8859_1 = 'ISO-8859-1'
```

Comment:
Do we need a deprecated backward-compat alias here for the typoed constant, so that users who copied the b0rked spelling don't break?

Reply:
Users who copied the b0rked spelling are already broken since any API request with the wrong encoding would fail.

Reply:
OK, LGTM.